[PATCH] parisc: __kernel_time_t is always long

From: Geert Uytterhoeven <geert@linux-m68k.org>

__kernel_time_t is always long on PA-RISC, irrespective of CONFIG_64BIT

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 include/asm-parisc/posix_types.h |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/include/asm-parisc/posix_types.h
+++ b/include/asm-parisc/posix_types.h
@@ -24,13 +24,12 @@ typedef int			__kernel_daddr_t;
 typedef unsigned long		__kernel_size_t;
 typedef long			__kernel_ssize_t;
 typedef long			__kernel_ptrdiff_t;
-typedef long			__kernel_time_t;
 #else
 typedef unsigned int		__kernel_size_t;
 typedef int			__kernel_ssize_t;
 typedef int			__kernel_ptrdiff_t;
-typedef long			__kernel_time_t;
 #endif
+typedef long			__kernel_time_t;
 typedef char *			__kernel_caddr_t;
 
 typedef unsigned short		__kernel_uid16_t;
